public/scripts/project/project.js

```diff
@@ -2,6 +2,7 @@ define(function(require) {
   var Constants = require("constants");
   var Remote = require("../../project/remote");
   var Metadata = require("../../project/metadata");
+  var Lock = require("../../project/lock");
```

Comment:
`require("lock");` is enough, based on what you did in your `paths` in the require config above.